flexboxAPIspecV2.md

```diff
@@ -0,0 +1,783 @@
+
+-->
+
+Flexbox Layout for WinUI 
+===
+
+# Background
+
+<!-- TEMPLATE
+    Use this section to provide background context for the new API(s) 
+    in this spec. Try to briefly provide enough information to be able to read
+    the rest of the document.
+
+    This section and the appendix are the only sections that likely
+    do not get copied to DMC; they're just an aid to reading this spec.
+
+    For example this is a place to provide a brief explanation of some dependent
+    area, just explanation enough to understand this new API, rather than telling
+    the reader "go read 100 pages of background information posted at ...".
+
+    For example this section is a place to explain why you're adding this new API rather than
+    using an existing related API.
+
+    For a simple example see the spec for the UIElement.ProtectedCursor property
+    (https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/UIElement/ElementCursor.md)
+    which has some of the thinking about how this Xaml API relates to existing
+    Composition and WPF APIs. This is interesting background both for the current reader
+    and the future reader trying to understand why we designed it this way,
+    but not the kind of information
+    that would land on DMC.
+-->
+
+
+Flexbox Layout is designed to provide an efficient way to layout, align, and distribute space among items within a container, specifically when the size of the device is either unknown or dynamic.
+
+Flexbox is a widely used layout schematic in web development, it allows developers to quickly create complex layouts with a standardized system. By introducing Flexbox to WinUI we can increase the layout choices developers have and cater to those who desire to use this popular layout in their WinUI applications.
+
+More on Flexbox available here: https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flexbox-properties
+
+
+# Remarks
+
+<!-- TEMPLATE
+    (Optional)
+
+    All APIs have a page on DMC, some APIs or groups of APIs have an additional high level,
+    conceptual page (called a "how-to" page). This section can be used for that content.
+
+    For example, there are several Xaml controls for different forms of text input,
+    each with an API page, and then there's also a conceptual page that
+    discusses them collectively
+    (https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/text-controls).
+
+    Another way to use this section is as a draft of a blog post that introduces the new feature.
+
+    Sometimes it's difficult to decide if text belongs on a how-to page or an API page.
+    It's not important to make a final decision on that in this spec; we can always
+    adjust it when copying to DMC.
+-->
+
+WinUI Offers several Layout options, such as StackLayout and Flowlayout. Usage of Flexbox Layout will behave similarly and interact by wrapping your desired elements in a Layout Control.
+
+#### Flex Containers vs Flex Items
```

Comment:
```suggestion
# Remarks
## Flex Containers vs Flex Items
```

This and the part about CSS below looks like it wants to be part of the "Conceptual pages" section in the spec template. Good idea and good topic. 

There should be some more overview content ahead of that generally describing what Flex is (target audience being Xaml developers; this section will end up on docs.microsoft.com). The goal is to enable someone to learn enough that they can comprehend what's being said in the rest of the doc.